src/seedu/addressbook/commands/DeleteCommand.java

```diff
@@ -29,8 +32,15 @@ public DeleteCommand(int targetVisibleIndex) {
     public CommandResult execute() {
         try {
             final ReadOnlyPerson target = getTargetPerson();
-            addressBook.removePerson(target);
-            return new CommandResult(String.format(MESSAGE_DELETE_PERSON_SUCCESS, target));
+            System.out.println("Enter 'yes' to continue:");
+            Scanner SCANNER = new Scanner(System.in);
```

Comment:
Should not have UI interfacing in your logic classes. You should make use of the I/O parts of Address Book to do that for you. You can refer to the Separation of Concerns principle (https://nus-cs2103-ay1819s2.github.io/cs2103-website/se-book-adapted/chapters/principles.html).